samwise.js

```diff
@@ -6,22 +6,22 @@
  */
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.